src/renderer.zig

```diff
@@ -199,9 +212,9 @@ pub fn renderWorld(world: *World, ambientLight: Vec3f, skyColor: Vec3f, playerPo
 	const meshes = mesh_storage.updateAndGetRenderChunks(world.conn, playerPos, settings.renderDistance);
 
 	gpu_performance_measuring.startQuery(.chunk_rendering_preparation);
-	const direction = crosshairDirection(game.camera.viewMatrix, lastFov, lastWidth, lastHeight);
+	const direction = crosshairDirection(baseViewMatrix, lastFov, lastWidth, lastHeight);
```

Comment:
Is there a reason we are using `crosshairDirection `instead of just `game.camera.direction`? `game.camera.direction` seems to do the same without having to extract the direction from the view matrix.

Reply:
#323 
Try moving the crosshair in menu mode. Check if block selection is still accurate after removing this code.